qiskit_machine_learning/kernels/quantum_kernel.py

```diff
@@ -90,6 +111,87 @@ def quantum_instance(
         else:
             self._quantum_instance = quantum_instance
 
+    @property
+    def free_parameters(self) -> Union[ParameterVector, Sequence[Parameter]]:
+        """Return the vector of free parameters."""
+        return self._free_parameters
+
+    @free_parameters.setter
+    def free_parameters(self, free_params: Union[ParameterVector, Sequence[Parameter]]):
+        """Sets the free parameters"""
+        self._free_param_binds = {free_params[i]: free_params[i] for i, _ in enumerate(free_params)}
+        self._free_parameters = free_params
+
+    def assign_free_parameters(
+        self, values: Union[Mapping[Parameter, float], Sequence[float]]
```

Comment:
I have updated the float hint to ParameterValueType. I'm not sure we want to provide an inplace option here, since we are essentially always spawning a copy of the unbound_feature_map when we assign parameters.

I don't believe QuantumCircuit provides a mechanism to bind and re-bind parameters in place?

Reply:
That's right, you can't re-bind parameters, once they are bound they are excluded from the list of parameters.